test/cql-pytest/test_describe.py

```diff
@@ -535,8 +542,9 @@ def test_desc_udf_uda(cql, test_keyspace, scylla_only):
 # Example: caching = {'keys': 'ALL', 'rows_per_partition': 'ALL'}
 # Reproduces #14895
 # The test is marked scylla_only because it uses a Scylla-only property "cdc".
-# xfail_tablets due to https://github.com/scylladb/scylladb/issues/16317
-def test_whitespaces_in_table_options(cql, test_keyspace, scylla_only, xfail_tablets):
+# xfails with tablets, due to https://github.com/scylladb/scylladb/issues/16317
+@pytest.mark.parametrize("test_keyspace", [pytest.param("tablets", marks=[pytest.mark.xfail]), "vnodes"], indirect=True)
```

Comment:
Again, please add "reason" for the xfail.

Reply:
Fixed in v2.